soh/soh/Enhancements/TimeDisplay/TimeDisplay.cpp

```diff
@@ -0,0 +1,255 @@
+        for (auto& check : activeTimers) {
+            if (check.timeID == timeID) {
+                activeTimers.erase(activeTimers.begin() + index);
+                return;
+            }
+            index++;
+        }
+    }
+}
+
+void TimeDisplayWindow::Draw() {
+    if (!gPlayState) {
+        return;
+    }
+    if (!CVarGetInteger(CVAR_WINDOW("TimeDisplayEnabled"), 0)) {
+        return;
+    }
+
+    ImGui::PushStyleColor(ImGuiCol_WindowBg, windowBG);
+    ImGui::PushStyleColor(ImGuiCol_Border, ImVec4(0, 0, 0, 0));
+    ImGui::PushStyleVar(ImGuiStyleVar_WindowRounding, 4.0f);
+
+    ImGui::Begin("TimerDisplay", nullptr, ImGuiWindowFlags_AlwaysAutoResize |
+            ImGuiWindowFlags_NoNav |
+            ImGuiWindowFlags_NoFocusOnAppearing |
+            ImGuiWindowFlags_NoResize |
+            ImGuiWindowFlags_NoDocking |
+            ImGuiWindowFlags_NoTitleBar |
+            ImGuiWindowFlags_NoScrollWithMouse |
+            ImGuiWindowFlags_NoScrollbar);
+    ImGui::SetWindowFontScale(fontScale);
+	if (activeTimers.size() == 0) {
+		ImGui::Text("No Enabled Timers...");
+	} else {
+		ImGui::BeginTable("Timer List", 2, ImGuiTableFlags_NoClip);
+		for (auto& timers : activeTimers) {
+			ImGui::PushID(timers.timeID);
+            TimeDisplayGetTimer(timers.timeID);
+            ImGui::TableNextColumn();
+            ImGui::Image(textureDisplay, ImVec2(16.0f * fontScale, 16.0f * fontScale));
+            ImGui::TableNextColumn();
+            
+            if (timeDisplayTime != "-:--") {
+                char* textToDecode = new char[timeDisplayTime.size() + 1];
+                textToDecode = std::strcpy(textToDecode, timeDisplayTime.c_str());
+                size_t textLength = timeDisplayTime.length();
+                uint16_t textureIndex = 0;
+            
+                for (size_t i = 0; i < textLength; i++) {
+                    ImVec2 originalCursorPos = ImGui::GetCursorPos();
+                    if (textToDecode[i] == ':' || textToDecode[i] == '.') {
+                        textureIndex = 10;
+                    } else {
+                        textureIndex = textToDecode[i] - 48;
+                    }
+                    if (textToDecode[i] == '.') {
+                        ImGui::SetCursorPosY(ImGui::GetCursorPosY() + (8.0f * fontScale));
+                        ImGui::Image(Ship::Context::GetInstance()->GetWindow()->GetGui()->GetTextureByName(digitList[textureIndex].first),
+                            ImVec2(8.0f * fontScale, 8.0f * fontScale), ImVec2(0, 0.5f), ImVec2(1, 1), textColor, ImVec4(0, 0, 0, 0));
+                    } else {
+                        ImGui::Image(Ship::Context::GetInstance()->GetWindow()->GetGui()->GetTextureByName(digitList[textureIndex].first),
+                            ImVec2(8.0f * fontScale, 16.0f * fontScale), ImVec2(0, 0), ImVec2(1, 1), textColor, ImVec4(0, 0, 0, 0));
+                    }
+                    ImGui::SameLine(0, 0);
+                }
+            }
```

Comment:
Some weird indentation formatting going on here

Reply:
Not sure where you are referring to here. Looks indented correctly in my editor.

Reply:
sorry the capture doesnt really show it but compared to the surrounding code its not got enough indentation as a whole block

Reply:
![image](https://github.com/user-attachments/assets/9bae34db-92f1-4358-a27e-0223768fcb4b)
Perhaps this will clarify what inspect was talking about.

Reply:
I think this is happening because some of your stuff is using tabs where it's using spaces for whitespace elsewhere. I think the spaces are our standard, so you should do a find and replace for tabs to 4 spaces.

Reply:
makes more sense, I just did an auto-format which should hopefully fix that on my next commit.